src/webui/api/searchcontroller.cpp

```diff
@@ -82,7 +82,7 @@ void SearchController::startAction()
     activeSearches.insert(id);
     session->setData(QLatin1String("activeSearches"), QVariant::fromValue(activeSearches));
 
-    const int hour = 3600000;
+    static const int hour = 3600000;
```

Comment:
This isn't needed IMO, marking it `static` will make it reside in memory at all times.

Reply:
>But won't this variable only get initialized once anyway?

Yes, but also that the same memory/object of MAX_CONCURRENT_SEARCHES will be shared by different `SearchController` instances (not that you really need them anyway...).